src/signals/incident/components/IncidentPreview/index.js

```diff
@@ -136,12 +115,13 @@ const IncidentPreview = ({ incident, preview }) => (
         visibleEntries.length > 0 && (
           <Section hasHeading={hasHeading} key={section}>
             <Header>
-              {sectionHeading || <div />}
-              <LinkContainer absolutePosLink={!hasHeading}>
-                <AscLink as={Link} to={`/incident/${section}`} variant="inline">
-                  Wijzigen
-                </AscLink>
-              </LinkContainer>
+              {sectionHeadingLabel ? (
```

Comment:
You can use the `sectionHeadingLabel && (` construct.
And don't you want to include the `<Header>` within this condition?